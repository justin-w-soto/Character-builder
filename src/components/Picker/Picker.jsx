import React from 'react'

export const Picker = () => {
    return (
        <div>
            
        </div>
    )
}
