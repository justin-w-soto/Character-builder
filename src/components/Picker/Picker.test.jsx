import { render, screen } from '@testing-library/react';
import Picker from './Picker.jsx';
it('should do the thing', () => {
    const { container } = render(<Picker />);

    const selectHead = screen.getByLabelText(/Head/);
    const selectMiddle = screen.getByLabelText(/Middle/);
    const selectPants = screen.getByLabelText(/Pants/);
    const input = screen.getByLabelText(/New Catchphrase:/);

    expect(selectHead).toBeInTheDocument();
    expect(selectMiddle).toBeInTheDocument();
    expect(selectPants).toBeInTheDocument();
    expect(input).toBeInTheDocument();
    expect(container).toMatchSnapshot();
})