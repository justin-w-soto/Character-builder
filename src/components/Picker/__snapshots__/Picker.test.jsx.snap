// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should do the thing 1`] = `
<div>
  <label>
    Head
    <select>
      <option>
        alien
      </option>
      <option>
        fish
      </option>
      <option>
        hooman
      </option>
    </select>
  </label>
  <label>
    Middle
    <select>
      <option>
        cool
      </option>
      <option>
        fancy
      </option>
      <option>
        hooman
      </option>
    </select>
  </label>
  <label>
    Pants
    <select>
      <option>
        bread
      </option>
      <option>
        dawg
      </option>
      <option>
        mane
      </option>
    </select>
  </label>
  <label>
    New Catchphrase:
    <input
      type="text"
      value=""
    />
  </label>
  <button
    type="button"
  >
    Submit
  </button>
</div>
`;
